documentation/IDTA-01001/modules/ROOT/pages/Spec/IDTA-01001_Metamodel_Common.adoc

```diff
@@ -303,7 +303,10 @@ Identifier of a supplemental semantic definition of the element called supplemen
 
 
 ====
-Note: it is recommended to use an external reference.
+Note: it is recommended to use an external reference. This is even recommended in the case a xref:IDTA-01001_Metamodel_Designators.adoc#ConceptDescription[ConceptDescription] exists. Otherwise lookup of the corresponding concept description may fail.
+
+In future releases a constraint may be added that makes is mandatory to use external references.
```

Comment:
```suggestion
In future releases a constraint may be added that makes it mandatory to use external references.
```

Small typo

Reply:
What is implied here, but not directly stated, is that we do not want the user to reference a "local" ConceptDescription that for example is shipped with an AASX package, right?

I fear that this formulation will create more AASX packages where constraint AASd-123 is violated: 

> Constraint AASd-123: For model references, i.e. References with Reference/type = ModelReference, the value of Key/type of the first key of Reference/keys shall be one of AasIdentifiables.

The reason for this is that someone who manually creates an AAS sees that one should refer to a `ConceptDescription` with an `ExternalReference` and do so, even if the referenced `ConceptDescription` is located inside the model and therefore needs a `ModelReference`.